### PR TITLE
Fix font rendering problem with og images for Korean pages

### PR DESCRIPTION
**Pull request by jcha0713**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

This pull request resolves a font rendering issue with Open Graph images that are generated for Korean pages.
I discovered that the matching Korean fonts do not exist which caused this error:

Screenshot_2023-02-21_at_10 23 58

I added Noto Sans KR fonts and the result is:

getting-started

### CODE REVIEW

**Review by Yan-Thomas — APPROVED** [review]
Nice catch @jcha0713, thanks for the fix! I'll let @delucis have a final look, but LGTM!

**Review by delucis — APPROVED** [review]
Awesome — thank you for fixing this @jcha0713!

**Pull request merged**
